translations/en-us.yaml

```diff
@@ -31,3 +31,15 @@ oss-components:
   select:
     search: Search
     placeholder: Select
+  upload-area:
+    drop_file:
+      default: Drop your file here, or
+      disabled: Drop your file here, or browse
+    browse: "browse"
+    errors:
+      filetype:
+        title: Upload error
+        description: The type of the file you uploaded is not allowed.
+      filesize:
+        title: Upload error
+        description: "The file you uploaded is too heavy. Maximum file size: {max_filesize}"
```

Comment:
Same here?

Reply:
I tend to quote them when i use a variable in the trad but it's not mandatory